deploying-daml-pathway.json

```
@@ -1,9 +1,9 @@
 {
-    "title": "Deploying with DAML",
+    "title": "Deploying DAML on Production Ledgers",
     "private": false,
     "courses": [
       {
-        "external_link": "TODO",
+        "external_link": "https://github.com/digital-asset/daml-on-fabric/",
```

```suggestion
        "external_link": "https://www.daml.com/learn/deploying-daml/daml-on-fabric/",
```

This should be the link for the daml.com/learn. We take the path after `courses` from the URL where the course will be located on katacoda.com/daml/ account (or from your URL, that part is the same, https://www.katacoda.com/anthonylusardi-da/courses/deploying-daml/daml-on-fabric)